OpenRA.Game/Scripting/ScriptObjectWrapper.cs

```diff
@@ -44,6 +61,13 @@ protected void Bind(IEnumerable<object> clrObjects)
 			}
 		}
 
+		protected void Unbind(Type targetType)
+		{
+			foreach (var m in members)
+				if (targetType == m.Value.Target.GetType())
+					members.Remove(m.Key);
```

Comment:
I was going to note that `Remove` inside a foreach would be invalid, but apparently that's ok for dictionary these days. Neat.

Reply:
#20355 not on mono it seems